Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3.9-slim-bullseye as base
+
+#Common needs
+#Set Working Directory
+WORKDIR /todo_app
+#install Poetry
+RUN pip install poetry
+#copy the config files
+COPY pyproject.toml poetry.toml poetry.lock ./
+#install dependencies
+RUN poetry install
+#Copy project files
+COPY . /
+#set startup and launch
+#working port
+EXPOSE 5000
+
+FROM base as Development
+
+
+# Development Specific needs here
+CMD ["poetry", "run", "flask", "run"]    
```

Comment:
You need --host 0.0.0.0 in your flask run command for it to listen to external requests. Same idea as including --bind 0.0.0.0 in your gunicorn command.

Reply:
amendment made